### PR TITLE
Remove gravitational/configure dependency

### PR DESCRIPTION
**Pull request by zmb3**
This repo hasn't been updated in 7 years and is not up to our current quality or security standards. In addition, we only leveraged a single function from it.

Updates #5685

### CODE REVIEW

**Comment by zmb3** [comment]
FYI @jentfoo - I think we should archive gravitational/configure once this is backported.

**Review by gzdunek — APPROVED** [review]

File: lib/utils/utils.go [hunk 1]
```diff
@@ -322,6 +323,31 @@ func IsValidHostname(hostname string) bool {
 	return true
 }
 
+// IsValidUnixUser checks if a string represents a valid
+// UNIX username.
+func IsValidUnixUser(u string) bool {
```
Comment (codingllama) [inline lib/utils/utils.go:328]: Are the tests in the source repo? Could we add a few here?

**Review by codingllama — APPROVED** [review]

**Comment by codingllama** [comment]
Dependency cleanups are always welcome, thank!

File: lib/utils/utils.go (hunk 1 above)
Comment (zmb3) [reply lib/utils/utils.go:328]: Tests? We didn't write tests 7 years ago! 😂

**Pull request merged**